templates/dashboard.html

```diff
@@ -25,18 +25,20 @@ <h2 class="mdl-card__title-text">Registrierte Karten</h2>
 
 	  <div class="mdl-card__media">
 		{{ if .Cards }}
-		<table class="mdl-data-table mdl-js-data-table">
+		<table id="cardtable" class="mdl-data-table mdl-js-data-table">
 		  <thead>
 			<tr>
 				<th class="mdl-data-table__cell--non-numeric">UID</th>
 				<th class="mdl-data-table__cell--non-numeric">Bezeichnung</th>
+				<th class="mdl-data-table__cell--non-numeric">Bearbeiten</th>
 			</tr>
 		  </thead>
 		  <tbody>
             {{ range .Cards }}
-			<tr>
+			<tr carduid="{{ printf "%x" .ID }}">
 				<td class="mdl-data-table__cell--non-numeric">{{ printf "%x" .ID }}</td>
 				<td class="mdl-data-table__cell--non-numeric">{{ .Description }}</td>
+				<td class="mdl-data-table__cell--non-numeric"><a href="edit_card.html?uid={{ printf "%x" .ID }}">edit</a></td>
```

Comment:
können wir "edit card" in 'nen separaten PR auslagern? Das ist 'ne ganz andere Art von Funktionalität :))